src/Library/demos/Frame/main.blp

```diff
@@ -0,0 +1,188 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: _("Frame");
+  description: _("A widget that surrounds its child with a decorative frame");
+
+  Box {
+    orientation: vertical;
+    halign: center;
+
+    Box {
+      orientation: horizontal;
+      halign: center;
+      spacing: 18;
+      margin-bottom: 18;
+      homogeneous: false;
+
+      Picture without_frame {
+        width-request: 180;
+        height-request: 180;
+      }
+
+      Separator {
+        margin-start: 6;
+        margin-end: 6;
+      }
+
+      Frame {
+
+        Picture with_frame {
+          width-request: 180;
+          height-request: 180;
+        }
+      }
+    }
+
+    Box {
+      homogeneous: true;
+      margin-bottom: 24;
+      halign: center;
+      spacing: 108;
```

Comment:
This is also considered an arbitrary number

please use box layout for centering
ask if you need help